docs/modules/data-structures/pages/vector-collections.adoc

```diff
@@ -0,0 +1,679 @@
+
+For similarity search, use the `searchAsync` method of the `VectorCollection`.
+
+In a single index vector collection, specifying the name of the index to search is not required.
+However, for a multi-index vector collection, specifying the name of the index to search in is necessary.
+
+Example for single-index vector collection:
+[tabs]
+====
+Java::
++
+--
+[source,java]
+----
+CompletionStage<SearchResults<String, String>> results = vectorCollection.searchAsync(
+        VectorValues.of(new float[] {0f, 0f, 0.2f, -0.3f, 1.2f, -0.5f}),
+        SearchOptions.builder()
+            .limit(5)
+            .includeVectors()
+            .includeValue()
+            .build()
+);
+----
+--
+Python::
++
+--
+[source,python]
+----
+results = vectorCollection.search_near_vector(
+    Vector("", Type.DENSE, [0, 0, 0.2, -0.3, 1.2, -0.5]),
+    limit=5,
+    include_value=True,
+    include_vectors=True,
+)
+----
+--
+====
+
+Example for multi-index vector collection:
+[tabs]
+====
+Java::
++
+--
+[source,java]
+----
+CompletionStage<SearchResults<String, String>> results = vectorCollection.searchAsync(
+        VectorValues.of("glove-index", new float[] {0f, 0f, 0.2f, -0.3f, 1.2f, -0.5f}),
+        SearchOptions.builder()
+            .limit(5)
+            .includeVectors()
+            .includeValue()
+            .build()
+);
+----
+--
+Python::
++
+--
+[source,python]
+----
+results = vectorCollection.search_near_vector(
+    Vector("glove-index", Type.DENSE, [0, 0, 0.2, -0.3, 1.2, -0.5]),
+    limit=5,
+    include_value=True,
+    include_vectors=True,
+)
+----
+--
+====
+
+=== Similarity search options
+Search parameters are passed as an argument `searchOptions` to the searchAsync method.
+
+.Search options
+[cols="1,2,1",options="header"]
+|===
+|Option|Description|Default
+
+|limit
+|How many results should be returned in search result
+|`10`
+
+|includeValue
+|Include the user value in the search result.
+|`FALSE`
+
+
+|includeVectors
+|Include the vectors values in the search result.
+|`FALSE`
+
+|hints
+|Extra hints for the search.
+|`NULL`
+
+|===
+
+
+.Available hints
+[cols="1,2",options="header"]
+|===
+|Hint|Description
+
+|partitionLimit
+|Number of results to fetch from each partition.
+
+|memberLimit
+|Number of results to fetch from member in 2-stage search.
+
+|singleStage
+|Force use of 1-stage search.
+
+|===
+
+[tabs]
+====
+Java::
++
+--
+[source,java]
+----
+var options = SearchOptions.builder()
+                .limit(10)
+                .includeValue()
+                .includeVectors()
+                .hint("partitionLimit", 1)
+                .build();
+----
+--
+====
+
+WARNING: Hints allow fine-tuning some aspects of search execution but are subject to change and may be removed in future versions.
+
+== Manage collection
+
+=== Optimize collection
+
```

Comment:
You can optimize a collection to improve performance, or clear a collection to remove all vectors and values.

Reply:
Sorry, i don't understand the remark